src/ChannelServer/World/SkillHandler.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Melia.Channel.Network;
+using Melia.Shared.Data.Database;
+using Melia.Shared.World;
+
+namespace Melia.Channel.World
+{
+	interface ITargetedSkillHandler
```

Comment:
You should always use appropriate access modifiers, which in this case would be `public` in my opinion. If we are going to add plugins in the future, we might want to be able to modify skill behavior from them, in which case we'll need access to these classes and interfaces.